queries.sql

```diff
@@ -8,3 +8,70 @@ SELECT (name, escape_attempt) FROM animals WHERE weight<10.5;
 SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals where weight>=10.4 AND weight<=17.3;
+
+--update species column to be unspecified;
+begin transaction;
+UPDATE animals
+SET species='unspecified';
+
+--roll back the change and verify that species columns went back to the state before transaction.
+ROLLBACK;
+
+--Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon';
+
+--Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals
+SET species='pokemon' WHERE species IS null;
+
+--Commit the transaction.
+COMMIT;
+
```

Comment:
- [ ] As pointed out earlier, please include queries to verify the change was made before committing and after committing